runtimes/pythonrt/runner/main.py

```diff
@@ -194,14 +194,12 @@ def Start(self, request: pb.runner.StartRequest, context: grpc.ServicerContext):
 
     def Execute(self, request: pb.runner.ExecuteRequest, context: grpc.ServicerContext):
         with self.lock:
-            call = self.activity_calls.pop() if self.activity_calls else None
```

Comment:
ah good i wanted to talk to you about that. if you do `pop` it will not work on retries.

Reply:
Any action item for me?

Reply:
nope